src/ActionModal/components/DelegationModal.vue

```diff
@@ -399,6 +406,34 @@ export default {
       update(data) {
         return data.balance || { amount: 0 }
       }
+    },
+    totalStaked: {
+      query: gql`
+        query overview($networkId: String!, $address: String!) {
+          overview(networkId: $networkId, address: $address) {
+            liquidStake
+            totalStake
```

Comment:
Wasn't the plan to actually remove `totalStake` from overview? Getting a bit confused right now

Reply:
but only in BalancesV2
we wanted to remove rewards from there

Reply:
ah ok, makes sense